<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowShowRobot</class>
 <widget class="QMainWindow" name="MainWindowShowRobot">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1114</width>
    <height>855</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Simox - VirtualRobot - Show Robot</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QFrame" name="frameViewer">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QGroupBox" name="groupBox">
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>201</width>
         <height>331</height>
        </rect>
       </property>
       <property name="title">
        <string>Reference Frames</string>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <widget class="QComboBox" name="comboBoxFrames">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>40</y>
          <width>181</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QDoubleSpinBox" name="spinBoxX">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>90</y>
          <width>76</width>
          <height>31</height>
         </rect>
        </property>
        <property name="minimum">
         <double>-1000.000000000000000</double>
        </property>
        <property name="maximum">
         <double>1000.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>10.000000000000000</double>
        </property>
       </widget>
       <widget class="QDoubleSpinBox" name="spinBoxY">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>130</y>
          <width>76</width>
          <height>31</height>
         </rect>
        </property>
        <property name="minimum">
         <double>-1000.000000000000000</double>
        </property>
        <property name="maximum">
         <double>1000.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>10.000000000000000</double>
        </property>
       </widget>
       <widget class="QDoubleSpinBox" name="spinBoxZ">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>170</y>
          <width>76</width>
          <height>31</height>
         </rect>
        </property>
        <property name="minimum">
         <double>-1000.000000000000000</double>
        </property>
        <property name="maximum">
         <double>1000.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>10.000000000000000</double>
        </property>
       </widget>
       <widget class="QDoubleSpinBox" name="spinBoxRoll">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>210</y>
          <width>76</width>
          <height>31</height>
         </rect>
        </property>
        <property name="minimum">
         <double>0.000000000000000</double>
        </property>
        <property name="maximum">
         <double>3.140000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.010000000000000</double>
        </property>
       </widget>
       <widget class="QDoubleSpinBox" name="spinBoxPitch">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>250</y>
          <width>76</width>
          <height>31</height>
         </rect>
        </property>
        <property name="minimum">
         <double>0.000000000000000</double>
        </property>
        <property name="maximum">
         <double>3.140000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.010000000000000</double>
        </property>
       </widget>
       <widget class="QDoubleSpinBox" name="spinBoxYaw">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>290</y>
          <width>76</width>
          <height>31</height>
         </rect>
        </property>
        <property name="minimum">
         <double>0.000000000000000</double>
        </property>
        <property name="maximum">
         <double>3.140000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.010000000000000</double>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>90</y>
          <width>67</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>X</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>130</y>
          <width>67</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Y</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>170</y>
          <width>67</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Z</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>210</y>
          <width>67</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Roll</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>250</y>
          <width>67</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Pitch</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>290</y>
          <width>67</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Yaw</string>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>350</y>
         <width>181</width>
         <height>241</height>
        </rect>
       </property>
       <property name="title">
        <string>Support Phase</string>
       </property>
       <widget class="QRadioButton" name="radioButtonLeftSupport">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>30</y>
          <width>121</width>
          <height>26</height>
         </rect>
        </property>
        <property name="text">
         <string>Left Support</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="radioButtonRightSupport">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>70</y>
          <width>141</width>
          <height>26</height>
         </rect>
        </property>
        <property name="text">
         <string>Right Support</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="radioButtonDualSupport">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>110</y>
          <width>141</width>
          <height>26</height>
         </rect>
        </property>
        <property name="text">
         <string>Dual Support</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1114</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
